website/content/docs/enterprise/sentinel/index.mdx

```diff
@@ -6,7 +6,13 @@ description: An overview of how Sentinel interacts with Vault Enterprise.
 
 # Overview
 
--> **Note**: This feature requires [Vault Enterprise Plus](https://www.hashicorp.com/products/vault/).
+<Note>
+
+Sentinel requires [Vault Enterprise
+Plus](https://www.hashicorp.com/products/vault/pricing/) license, or an [HCP
+Vault Plus](/hcp/docs/vault/tiers-and-features#plus-tier) cluster.
+
+</Note>
```

Comment:
Whilst this block is being updated, would you be open to taking a look at https://github.com/hashicorp/vault/issues/18525, which I filed many months ago, calling out inconsistencies in the self-hosted Enterprise tier in various parts of the docs, including this one?

Reply:
I created a [Jira ticket](https://hashicorp.atlassian.net/browse/SPE-322) to follow up on this.  The issue was assigned to someone who left the company that it was left opened. Sorry!